public/app/features/playlist/utils.ts

```diff
@@ -71,6 +71,8 @@ export async function loadDashboards(items: PlaylistItemUI[]): Promise<PlaylistI
 
 export function getDefaultPlaylist(): Playlist {
   return {
+    kind: 'Playlist',
+    apiVersion: '',
```

Comment:
Are these hand-rolled? I think the version should use the preferred version (though not sure if there's already a way to get that dynamically from the API):
```suggestion
    apiVersion: 'v0alpha1',
```